Doc/library/inspect.rst

```diff
@@ -765,6 +765,16 @@ function.
        .. versionadded:: 3.10
           ``globalns`` and ``localns`` parameters.
 
+   .. classmethod:: Signature.from_code(co)
+
+       Return a :class:`Signature` (or its subclass) object
+       for a given :class:`code object <types.CodeType>` ``co``.
+
+       Since code objects do not know anything
+       about default values or annotations,
+       they will always be ommited from the signature.
```

Comment:
You can suggest using Signature.from_callable() when a function is available.

In from_callable() doc, you may mention from_code() but with a quick reminder about limitations (no default values nor annotations). Or maybe don't mention it there :-)